uplink/metainfo/kvmetainfo/buckets_test.go

```diff
@@ -213,17 +213,34 @@ func getBucketNames(bucketList storj.BucketList) []string {
 }
 
 func runTest(t *testing.T, test func(*testing.T, context.Context, *testplanet.Planet, *kvmetainfo.DB, streams.Store)) {
-	testplanet.Run(t, testplanet.Config{
-		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
-	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		db, streams, err := newMetainfoParts(planet)
+	runPlanet(t, func(t *testing.T, ctx context.Context, planet *testplanet.Planet) {
+		db, streams, err := newMetainfoParts(planet, newTestEncStore(TestEncKey))
 		require.NoError(t, err)
 
 		test(t, ctx, planet, db, streams)
 	})
+
+}
+
+func newTestEncStore(keyStr string) *encryption.Store {
+	key := new(storj.Key)
+	copy(key[:], keyStr)
+
+	store := encryption.NewStore()
+	store.SetDefaultKey(key)
+
+	return store
+}
+
+func runPlanet(t *testing.T, test func(*testing.T, context.Context, *testplanet.Planet)) {
```

Comment:
Why this helper? It doesn't look like it's adding anything much compared to using `testplanet.Run` directly.

Reply:
It used have more in it.. I kept it so that I wouldn't have to duplicate the testplanet config. I can remove it if you like - I don't feel strongly about it.

Reply:
Yeah, I would probably avoid it because it only saves two lines, without much benefit.